getting-started/create-cagg/create-cagg-basics.md

```diff
@@ -157,4 +162,4 @@ found in the [continuous aggregates docs][continuous-aggregates].
 [min]: https://www.postgresql.org/docs/current/tutorial-agg.html
 [first]: /api/:currentVersion:/hyperfunctions/first/
 [last]: /api/:currentVersion:/hyperfunctions/last/
-[cagg-policy]: /getting-started/create-cagg/create-cagg-policy/
\ No newline at end of file
+[cagg-policy]: /getting-started/create-cagg/create-cagg-policy/
```

Comment:
```suggestion
[cagg-policy]: /create-cagg/create-cagg-policy/
```
The `continuous-aggregates` link above also needs to be fixed. It should be `/timescaledb/:currentVersion:/how-to-guides/continuous-aggregates`. It's too far from the edited portion so I can't submit a comment on it.